### ISSUE TITLE
Can't Start/Stop Instances in Orchestrations

### ISSUE THREAD

**Issue by fribeiro1** [terraform-providers/terraform-provider-opc#107]
### Terraform Version
Terraform v0.11.1
provider.opc v1.0.1

### Affected Resource(s)
opc_compute_orchestrated_instance

### Terraform Configuration Files
```hcl
resource "opc_compute_orchestrated_instance" "SampleInstance" {
  desired_state = "active"
  name = "SampleOrchestration"
  instance {
    boot_order = [1]
    desired_state = "running"
    name = "SampleInstance"
    persistent = true
    shape = "oc3"
    storage {
      index = 1
      volume = "${opc_compute_storage_volume.SampleVolume.name}"
      volume = "SampleVolume"
    }
  }
}
```

### Debug Output
https://gist.github.com/fribeiro1/3fcc02f6536a051f0df0af8fd672249b

### Panic Output
N/A

### Expected Behavior
Instances in orchestrations can be started/stopped with Terraform like with the other tools (console, API and CLI).

### Actual Behavior
Terraform issues an "invalid or unknown key: desired_state" when the "desired_state" key is added to  instances in orchestrations.

Also note that the docs (https://www.terraform.io/docs/providers/opc/r/opc_compute_orchestrated_instance.html) incorrectly say that "Instance supports the arguments found in opc_compute_instance" even though the "desired_state" attribute is not supported.

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
None

### References
None

**Comment by scross01** [comment]
The `desired_state` is not supported within the instance block for orchestrated instances. Instead change the `desired_state` of the `opc_compute_orchestrated_instance` from `active` to `suspend` (this is the one exception, all other `opc_compute_instance` attributed are supported).

Note: The difference with this approach is you are suspending the orchestration rather than suspending the instance directly.  The Orchestration framework will then terminate the instance, and when setting the orchestration `desired_state` back to `active` the Orchestration framework recreates the instance.

**Comment by scross01** [comment]
To add to this, it looks like support for `desired_state` in the instance definition could be added, but the process to set it is a bit convoluted. 

First, why enable `desired_state` at the instance level as well as the orchestration level? - well there is a difference in behavior between how the Oracle Cloud terminates/suspends and recreates the instance when its done at the Orchestration level vs Instance level which affect the UUID of the instances. For most cases this isn't an issue, by if an instance needs a consistent UUID (e.g for application licensing) then having the UUID change when an instance is shutdown and restarted can be a problem. 

1. For `opc_compute_instance` resources, when setting the `desired_state` to `shutdown` the instance definition is retained in Oracle Cloud - the instance continues to appear in UI (in a shutdown state) and most importantly the instance retains the **same UUID** when set back to 'running'

2. For the `opc_compute_orchestrated_instance` resources, setting the `desired_state` of the Orchestration to `suspend` will stop (destroy) the child instance, and when setting the Orchestration back to `active` the orchestration recreates the instance with a **different UUID**

One caveat to setting the orchestration level `desired_state` to suspended is when the instance level `persistent` attribute is set. When the instance is set with `persistent = true` and the orchestration is in a suspended state, the instance remains in a running state (i.e. the orchestration does not destroy the persistent instances).  

This is where things get interesting, while the Orchestration is in a "suspended" state the orchestration object definitions can be modified, so as this point it is now _possible_ to update the instance level `desired_state` to `shutdown` (note: this option is disabled in the UI, but can be set by manually adding `"desired_state": "shutdown"` in the orchestration JSON via the UI).  The result is the same as when updating desired state of an instance created via a Launch Plan, the instance is shutdown, but the instance definition is retained and can be restarted with the same UUID.

So to summarize, the (proposed) process to shutdown/restart an orchestrated instance and retain the UUID is:

1. create a `opc_compute_orchestrated` instance (with a persistent boot volume) and with `persistent = true"` in the instance definition.
2. change the orchestration `desired_state = "suspend"` and re-apply
3. change the instance `desired_state = "shutdown"` and re-apply (_not currently supported by the provider, but can be manually set in the UI by modifying the orchestration JSON_)
4. The instance is now shutdown
5. change the instance `desired_state = "running"` and re-apply (_not currently supported by the provider, but can be manually set in the UI by modifying the the orchestration JSON_)
6. change the orchestration `desired_state = "active"` and re-apply

**Comment by fribeiro1** [comment]
Out of date.
